cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -187,6 +187,11 @@
      * Name of the property used to identify whether the dataset is local or not.
      */
     public static final String WORKFLOW_LOCAL_DATASET_PROPERTY = "workflow.local.dataset";
+
+    /**
+     * Configuration setting to provide extra jars for programs.
+     */
+    public static final String EXTRA_JARS = "extra.jars";
```

Comment:
this key is unclear what do mean by extra jars. It's better call it `program.container.dist.jars` (similar to `spark.yarn.dist.jars` in Spark).